modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -31,8 +31,8 @@ See link:https://access.redhat.com/documentation/en-us/openshift_container_platf
 +
 When you add a site such as `quay.io` to your allowlist, do not add a wildcard entry such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, then image downloads are denied when the initial download request is redirected to a host name such as `cdn01.quay.io`.
 
-. Managed clusters require Telemetry to be enabled to allow Red Hat to react more quickly to problems and better support our customers, as well as better understand how product upgrades impact clusters.  
-See link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
+. Managed clusters require Telemetry to be enabled to allow Red Hat to react more quickly to problems and better support our customers, as well as better understand how product upgrades impact clusters.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/support/remote-health-monitoring-with-connected-clusters#about-remote-health-monitoring[About remote health monitoring] for more information about how remote health monitoring data is used by Red Hat.
```

Comment:
4.8.2 is current default version on production env. Should the url be changed to https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html/support/remote-health-monitoring-with-connected-clusters?